tests/Microsoft.DotNet.Docker.Tests/TestData.cs

```diff
@@ -348,6 +348,28 @@ public static class TestData
             },
         };
 
+        private static readonly ProductImageData[] s_ReverseProxyTestData =
+{
+            new() {
+                Version = new ImageVersion(new Version(2,3), isPreview: true),
+                VersionFamily = V9_0,
+                OS = OS.AzureLinux30Distroless,
+                OSTag = "",
+                OSDir = OS.AzureLinux30Distroless,
+                Arch = Arch.Amd64,
+                SupportedImageRepos = DotNetImageRepo.Reverse_Proxy,
+            },
+            new() {
+                Version = new ImageVersion(new Version(2,3), isPreview: true),
+                VersionFamily = V9_0,
+                OS = OS.AzureLinux30Distroless,
+                OSTag = "",
+                OSDir = OS.AzureLinux30Distroless,
+                Arch = Arch.Arm64,
+                SupportedImageRepos = DotNetImageRepo.Reverse_Proxy
+            },
+        };
+
```

Comment:
nit: formatting + use collection expression.

```suggestion
        private static readonly ProductImageData[] s_ReverseProxyTestData =
        [
            new() {
                Version = new ImageVersion(new Version(2,3), isPreview: true),
                VersionFamily = V9_0,
                OS = OS.AzureLinux30Distroless,
                OSTag = "",
                OSDir = OS.AzureLinux30Distroless,
                Arch = Arch.Amd64,
                SupportedImageRepos = DotNetImageRepo.Reverse_Proxy,
            },
            new() {
                Version = new ImageVersion(new Version(2,3), isPreview: true),
                VersionFamily = V9_0,
                OS = OS.AzureLinux30Distroless,
                OSTag = "",
                OSDir = OS.AzureLinux30Distroless,
                Arch = Arch.Arm64,
                SupportedImageRepos = DotNetImageRepo.Reverse_Proxy
            },
        ];

```